src/mmw/js/src/draw/views.js

```diff
@@ -88,6 +89,7 @@ var SelectAreaView = Marionette.ItemView.extend({
             endpoint = $el.data('endpoint'),
             tableId = $el.data('tableid');
 
+        App.map.set('areaOfInterest', null);
```

Comment:
Ideally, this view would follow the design pattern of the other drawing tools, so that the revert layer logic is similarly self-contained.